README.md

```diff
@@ -256,8 +256,9 @@ There are two ways to achieve this, by using `sshuttle` or `xinetd`.
 2. Add entry to `/etc/hosts`
 
     ```
-    192.168.111.4 console-openshift-console.apps.ostest.test.metalkube.org console openshift-authentication-openshift-authentication.apps.ostest.test.metalkube.org api.ostest.test.metalkube.org prometheus-k8s-openshift-monitoring.apps.ostest.test.metalkube.org alertmanager-main-openshift-monitoring.apps.ostest.test.metalkube.org kubevirt-web-ui.apps.ostest.test.metalkube.org oauth-openshift.apps.ostest.test.metalkube.org grafana-openshift-monitoring.apps.ostest.test.metalkube.org
-    ```
+    192.168.111.4 console-openshift-console.apps.ostest.test.metalkube.org console openshift-authentication-openshift-authentication.apps.ostest.test.metalkube.org prometheus-k8s-openshift-monitoring.apps.ostest.test.metalkube.org alertmanager-main-openshift-monitoring.apps.ostest.test.metalkube.org kubevirt-web-ui.apps.ostest.test.metalkube.org oauth-openshift.apps.ostest.test.metalkube.org grafana-openshift-monitoring.apps.ostest.test.metalkube.org
+    192.168.111.5 api.ostest.test.metalkube.org
```

Comment:
This works. But I wonder if it's better to recommend to take the IP addresses from `/etc/NetworkManager/dnsmasq.d/openshift-ostest.conf` on the dev-scripts machine. This will be never broken by dev-scripts changes.

You can still keep the addresses here as examples.